lib/appliance_console.rb

```diff
@@ -219,11 +220,11 @@ module ApplianceConsole
         MAC Address:       #{mac}
         Timezone:          #{timezone}
         Local Database:    #{ApplianceConsole::Utilities.pg_status}
-        EVM Database:      #{dbtype} @ #{dbhost}
-        Database/Region:   #{database} / #{region || 0}
+        EVM Database:      #{ApplianceConsole::DatabaseConfiguration.configured? ? "#{dbtype} @ #{dbhost}" : "not configured"}
```

Comment:
Should we store  `configured?` in a variable once, each time within this loop.  configured? checks for the database.yml each time it's called, right?